osi_sensordata.proto

```diff
@@ -297,4 +300,8 @@ message SensorData
     // object hypothesis and tracking.
     //
     optional FeatureData feature_data = 26;
+
+    // Host vehicle data measured by the sensors.
+    //
+    optional HostVehicleData host_vehicle = 27;
```

Comment:
@pmai can we add the time stamp as a DetectedEntityHeader to HostVehicleData? #463 
@jruebsam FYI

`    // General information about the \c HostVehicle .
    //
    optional DetectedEntityHeader host_vehicle_header = 27;

    // Host vehicle data measured by the sensors.
    //
    optional HostVehicleData host_vehicle = 28;`

Reply:
@pmai or should me define/add a specific header for host vehicle?